kotlin/src/main/com/looker/sdk/4.0/methods.kt

```diff
@@ -48,12 +50,13 @@ class LookerSDK(authSession: AuthSession) : APIMethods(authSession) {
      * POST /alerts/{alert_id}/follow -> Void
      */
     fun follow_alert(
-        alert_id: String,
-    ): SDKResponse {
+        alert_id: String
+    ) : SDKResponse {
```

Comment:
we need to automate the new linter script. Perhaps @tjbanghart has created a reformatter script for kotlin already?

Reply:
I have not but will create an issue to do so. Thank you for flagging. I can follow up with the lint change when that is complete.